website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -34,7 +34,9 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------- | ------------ |
 | `NO`             | `none`            | `none`        | `acl:write`  |
 
-### Payload Fields
+-> The corresponding CLI command is [`consul acl auth-method create`](https://www.consul.io/commands/acl/auth-method/create)
```

Comment:
Please remove `https://www.consul.io` from all of these links (as annoying as it will be to update every line). That will allow for these links to work as intended anywhere the site is run (locally, a preview environment, or production). As is, these links will only ever point to production (even if working locally or in a preview/staging environment).

```suggestion
-> The corresponding CLI command is [`consul acl auth-method create`](/commands/acl/auth-method/create)
```

Before you apply this change, wait for a response on my question below to Karl and Adam. I want you to only need to touch each line once more rather than go back multiple times.

Reply:
Made the necessary changes in one of my later commits